noxfile.py

```diff
@@ -180,3 +180,18 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+@nox.session
+def tag(session: nox.Session):
+    """
+    Check the core repo for new releases and create tags in ansible-documentation
+    """
+    install(session, req="tag")
+    args = list(session.posargs)
+
+    # If run without any arguments, default to "tag"
+    if not any(arg.startswith(("hash", "mantag", "new-tags", "tag")) for arg in args):
+        args.append("tag")
+
+    session.run("python", "hacking/tagger/tag.py", *args)
```

Comment:
Instead of making a list and mutating it, why not make an additional var just for those extra args?
```suggestion
    known_cli_args_present = any(arg.startswith(("hash", "mantag", "new-tags", "tag")) for arg in session.posargs)
    extra_cli_args = () if known_cli_args_present else ("tag",)

    session.run("python", "hacking/tagger/tag.py", *session.posargs, *extra_cli_args)
```

Reply:
I was matching the existing `pip-compile` session that does the same `args.append()`. I'm personally in favor of keeping things consistent. If we want to change it, we should change both, possibly as a separate effort from this.